audioplayer.cpp

```diff
@@ -0,0 +1,99 @@
+#include "audioplayer.h"
+
+vector<string> getFileNames(const string& folder) {
+    vector<string> names;
+    DIR* directory = opendir(folder.c_str());
+
+    if (directory == nullptr) {
+        abort();
+    }
+
+    struct dirent* ent;
```

Comment:
I don't know the syntax, but it looks like `struct` keyword is useless here

Reply:
yep, you're right